common/persistence/sql/sqlplugin/postgresql/session/session.go

```diff
@@ -62,6 +64,18 @@ func NewSession(
 	cfg *config.SQL,
 	resolver resolver.ServiceResolver,
 ) (*Session, error) {
+	if cfg.AuthPlugin != "" {
+		authPlugin, err := SQLAuth.LookupPlugin(cfg.AuthPlugin)
+		if err != nil {
+			return nil, err
+		}
+
+		cfg, err = authPlugin.GetConfig(context.TODO(), cfg)
```

Comment:
Please use timeout here as well.

Reply:
✅ Sorry, my Go is rusty - I added a new config field for specifying the timeout.